delivery_recipe/daily_standup.md

```diff
@@ -1,21 +1,27 @@
 # Daily Standup
 
-The daily standup is an important meeting that we hold every day during each sprint.
+The daily standup is an important meeting that we hold at the same time, same place, every day during each sprint.
 
 ## Who should attend?
 
-The meeting must be attended by the Delivery Team, the Service Delivery Manager and the Product Owner. Other people can join but only as observers.
+The meeting must be attended by the Delivery Team, the Service Delivery Manager and the Product Owner. Other people can join but only as observers, they don’t contribute.
 
 ## The purpose
 
-The reason we have a daily standup is due to the fast paced, ever changing nature of the work we deliver. The daily standup is the opportunity for the each member of the team to share the details of what they're working on, what they have planned for that day and to raise any problems they're encountering. The meeting also serves to communicate the progress with the Product Owner and to seek help if any members of the delivery team have encountered problems that need to be resolved.
+The reason we have a daily standup is due to the fast paced, ever changing nature of the work we deliver. The daily standup is the opportunity for each member of the team to share what they’ve completed and what they’re working on. It’s a great way to communicate progress, challenges with the rest of the team and with the Product Owner.
 
 ## The process
 
-Each member of the delivery team takes turns to discuss the stories they worked on the previous day, what they plan to work on today and to raise any issues that they’ve encountered.
+Each member of the delivery team takes turns to discuss the stories they worked on the previous day, what they plan to work on today and to raise any issues that they’ve encountered. The Delivery Manager will chair the meeting, making sure everyone speaks up, keeping everyone on topic and keeping the non-participants out of the conversations.
```

Comment:
'chair' makes it sound like they are in charge of a traditional meeting. Facilitate expresses their role better I think